### PR TITLE
Vic 923 enable topics in messages

### PR DESCRIPTION
**Pull request by tkuzynow**
Fixes #

## Proposed Changes

  -
  -
  -

### CODE REVIEW

File: src/test/java/de/caritas/cob/userservice/api/adapters/web/controller/UserControllerE2EIT.java [hunk 1]
```diff
@@ -1162,6 +1213,39 @@ private void givenAValidRocketChatInfoUserResponse(Consultant consultant) {
...
+    var roomsGetDTO = new RoomsGetDTO();
+    roomsGetDTO.setUpdate(new RoomsUpdateDTO[]{});
+    when(restTemplate.exchange(
+        Mockito.anyString(), eq(HttpMethod.GET),
+        any(HttpEntity.class), eq(RoomsGetDTO.class))
+    ).thenReturn(ResponseEntity.ok(roomsGetDTO));
+  }
+
+  private void givenAValidTopicServiceResponse() {
+    var roomsGetDTO = new RoomsGetDTO();
```
Comment (patric-dosch-vi) [inline src/test/java/de/caritas/cob/userservice/api/adapters/web/controller/UserControllerE2EIT.java]: is that RoomsGetDTO here needed?

File: src/test/resources/database/UserServiceDatabase.sql [hunk 2]
```diff
@@ -1159,3 +1164,13 @@ VALUES (1, '1da238c6-cd46-4162-80f1-bff74eafe77f', '473f7c4b-f011-4fc2-847c-ceb6
...
         '2020-10-08 09:03:47', '2020-10-08 09:03:48');
+
+INSERT INTO session (`id`, `user_id`, `consultant_id`, `consulting_type`, `message_date`,
+                     `postcode`, `agency_id`, `rc_group_id`, `rc_feedback_group_id`, `status`,
+                     `is_team_session`, `is_monitoring`, `create_date`, `update_date`,`main_topic_id`)
+VALUES (1215, '1da238c6-cd46-4162-80f1-bff74eafe77f', '34c3x5b1-0677-4fd2-a7ea-56a71aefd099', 0,
+        '2020-10-08 09:03:45', '12345', 1, 'DJrRTzFg8Ac2BqE8j', 'E6FohkNfEDPY9bRXc', 2, 0, 1,
+        '2020-10-08 09:03:45', '2020-10-08 09:03:45', 1);
+
+UPDATE SESSION set main_topic_id = 1
```
Comment (patric-dosch-vi) [inline src/test/resources/database/UserServiceDatabase.sql]: I understand, that we need a session entry with the main_topic_id. But why generating one with an insert and update another with new update? Why not generating a set of sessions with main topic?

**Review by patric-dosch-vi — DISMISSED** [review]
hey @tkuzynow, just two small comments.

File: src/test/java/de/caritas/cob/userservice/api/adapters/web/controller/UserControllerE2EIT.java (hunk 1 above)
Comment (tkuzynow) [reply src/test/java/de/caritas/cob/userservice/api/adapters/web/controller/UserControllerE2EIT.java]: no it's not. I've removed it, thanks for spotting this!

File: src/test/resources/database/UserServiceDatabase.sql (hunk 2 above)
Comment (tkuzynow) [reply src/test/resources/database/UserServiceDatabase.sql]: changed to separate insert group for sessions with topics

**Review by patric-dosch-vi — APPROVED** [review]

File: src/test/resources/database/UserServiceDatabase.sql (hunk 2 above)
Comment (patric-dosch-vi) [reply src/test/resources/database/UserServiceDatabase.sql]: thanks

**Review by adnanalicic — APPROVED** [review]

**Pull request merged**